### DISCUSSION TITLE
Not able to select Javascript, Defaults to Typescript

### DISCUSSION THREAD

**Discussion by Desdfortier** [remix-run/remix#8545] [Q&A] [answered]
Good day, I am trying to create a remix project with npx create-remix@latest, in the docs it show I can select the framework. However I don't get this selection.
1. Running the command in Powershell on a Windows 11 home machine.
2. Node Version : v20.10.0, NPM Version: 10.3.0

I have tried running the basic template as well and still no luck, how can I install Remix and use Javascript instead of Typescript

**Comment by kiliman** [comment, marked as answer]
Looks like this has been removed from `create-remix`

<img width="910" alt="image" src="https://github.com/remix-run/remix/assets/47168/0544f644-d51c-45f4-9678-07dd8cad4539">


https://github.com/remix-run/remix/pull/6887

**Reply by sergiodxa** [reply]
Yep, you need to find a template that uses JS instead of TS, then you can do:
```sh
npx create-remix@latest --template user/repo-with-js
```

**Reply by Desdfortier** [reply]
Thanks this has resolved the issue, there is a template reference in the docs in Remix to this. This is the Templates that is referenced 
npx create-remix@latest --template remix-run/remix/templates/remix-javascript

**Comment by ShaileshKV29** [comment]
I am still getting root.tsx and not root .jsx even after selecting the js template, please help

**Reply by kiliman** [reply]
Did you use this command?

```bash
npx create-remix@latest --template remix-run/remix/templates/remix-javascript
```
This is the template repo that it would use.

https://github.com/remix-run/remix/tree/main/templates/remix-javascript
